examples/applications/plot_model_complexity_influence.py

```diff
@@ -16,17 +16,21 @@
 
 We will model the complexity influence on three different estimators:
     - :class:`~sklearn.linear_model.SGDClassifier` (for classification data)
-      which implements stochastic gradient descent learning;
+      which implements stochastic gradient descent learning.
+      See example for comparison with different penalties:
+      :ref:`sphx_glr_auto_examples_linear_model_plot_sgd_penalties.py`
 
     - :class:`~sklearn.svm.NuSVR` (for regression data) which implements
-      Nu support vector regression;
+      Nu support vector regression. See example of its application:
+      :ref:`sphx_glr_auto_examples_svm_plot_svm_regression.py`
```

Comment:
This example might be a better fit for the doc-strings of [SVR](https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVR.html#sklearn.svm.SVR) and [NuSVR](https://scikit-learn.org/stable/modules/generated/sklearn.svm.NuSVR.html), and in the [Support Vector Machines user guide](https://scikit-learn.org/stable/modules/svm).

Reply:
Hi @virchan, thanks for the suggestion! I found an appropriate spot to add this example in the `SVR`, `NuSVR` doc-strings, as well as in the Support Vector Machines user guide. Please wait for my upcoming commits with these updates.